modules/programs/lieer.nix

```diff
@@ -288,6 +292,13 @@ in {
       programs.notmuch.new.ignore = [ "/.*[.](json|lock|bak)$/" ];
 
       home.file = listToAttrs (map configFile lieerAccounts);
+
+      # Lieer requires the maildir structure to exist before syncing will work.
+      # Normally this is done via 'gmi init', but 'gmi init' bails out if the
+      # config file alreasy exists.
+      home.activation.initLieer =
```

Comment:
An alternative that is used in some places it to create a `.keep` file. See e.g the dconf module:
https://github.com/nix-community/home-manager/blob/e13aa9e287b3365473e5897e3667ea80a899cdfb/modules/misc/dconf.nix#L79-L80

I don't know if that would work in this case, though.